content/en/blog/2022-08-26-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,213 @@
+
+The Flux CLI generates a single layer OCI image for storing things. As you can use some other tools to generate an OCI image with multiple layers in it, you can use the [Layer Selection](https://github.com/fluxcd/flux2/tree/main/rfcs/0003-kubernetes-oci#layer-selection) feature that Flux provides to select the layers you want to use in the OCI image. If the layer selector matches more than one layer, the first layer matching the specified media type will be used. Note that Flux requires that the OCI layer is compressed in the tar+gzip format.
+
+{{< imgproc twitter-ss Resize 400x >}}
+{{< /imgproc >}}
+
+{{< imgproc meme Resize 400x >}}
+{{< /imgproc >}}
+
+Today, we’ll leverage the OCI Repositories feature to apply Kyverno policies stored in an OCI registry into the Kubernetes cluster.
+
+First, we need to install Flux CLI, please see the [installation](https://fluxcd.io/docs/installation/) page for more details.
+
+Next, we should have a Kubernetes cluster running. We’ll be using [KinD](https://kind.sigs.k8s.io/docs/user/quick-start#configuring-your-kind-cluster) for this purpose.
+
+```shell
+kind create cluster
+```
+
+Once the cluster has been provisioned successfully, we need to install Flux components into it by simply running the command below:
+
+```shell
+$ flux bootstrap github \
+  --owner=developer-guy \
+  --repository=flux-kyverno-policies \
+  --path=clusters/local \
+  --personal
+```
+
+> ⚠️ **Note:** Don't forget to change the values with your own details!
+
+This command will install Flux and create necessary files for us and push them into the repository.
+
+Next, we should install Kyverno by using a GitOps approach with Flux. In order to do that, we use the following resources:
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: GitRepository
+metadata:
+  name: kyverno-controller
+  namespace: flux-system
+spec:
+  interval: 30m
+  url: https://github.com/kyverno/kyverno
+  ignore: |
+    /*
+    !/config/
+  ref:
+    semver: "1.x"
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: kyverno-controller
+  namespace: flux-system
+spec:
+  interval: 30m
+  sourceRef:
+    kind: GitRepository
+    name: kyverno-controller
+  serviceAccountName: kustomize-controller
+  path: ./config/release
+  prune: true
+  wait: true
+  timeout: 5m
+```
+
+Do not forget to check whether everything works fine before moving into the next steps:
+
+```shell
+$ flux get helmreleases kyverno
```

Comment:
There is no Helm release, please update the command and output to `flux get kustomizations`.

Reply:
done.